README.md

```diff
@@ -11,11 +11,29 @@ language that recognizes Church encoded structures.
 Before doing anything useful with Morganey you need to install
 [sbt][scala-sbt] first.
 
+### Samples ###
+
+For language samples and examples please visit [docs/samples/][samples-dir] directory. To execute a sample run:
+
+```console
+$ sbt "run <path-to-sample>"
+```
+
+from the root of the source code directory. For example, to run the 15 first fibonacci numbers sample run:
```

Comment:
"To print 15 first fibonacci sample"? What "printing a sample" is supposed to mean in this context? I'd replace it with `execute`. Let me do that.